controller/update.go

```diff
@@ -201,6 +201,16 @@ func (c controller) UpdateWithStrategy(ctx context.Context, req Request, opts Up
 	var addInProgress int64
 	var removeInProgress int64
 
+	// To keep n units alive during the update,
+	// we need to have just more than that in current units,
+	// and units we're allowed to create during the update.
+	if !(numTotal+opts.MaxGrowth > opts.MinAlive) {
```

Comment:
this allows me to do this:

```
inagoctl submit mygroup 5
instalctl start mygroup
inagoctl update mygroup --max-growth=5 --min-alive=9
```

Which clearly means, it cannot reduce the number of slices after the initial growth. I don't think this is what we want.